p1.py

```diff
@@ -0,0 +1,49 @@
+# Characters in phone corresponding to numbers from 2 to 9
+charsInPhone = [['a','b','c'],       # 2
```

Comment:
charsInPhone is a global variable, it is better to name it as `CHARS_IN_PHONE`. also I think Camel-Case is a java style hahaha. How about we use the python style.  For example [PEP8](https://www.python.org/dev/peps/pep-0008/)